bin/inx-chronicle/src/api/stardust/history/routes.rs

```diff
@@ -66,9 +68,54 @@ async fn transaction_history(
         })
         .collect::<Result<_, ApiError>>()?;
 
-    Ok(TransactionHistoryResponse {
-        items: transactions,
+    Ok(TransactionsPerAddressResponse {
         address,
+        items: transactions,
+        cursor,
+    })
+}
+
+async fn transactions_by_milestone_history(
+    database: Extension<MongoDb>,
+    Path(milestone_id): Path<String>,
```

Comment:
Sorry, I made a mistake here, this should actually be `milestone_index`, which we use throughout most of the API.

Reply:
Changed.